content/blog/2016/2016-07-04-ewm-alpha-version.adoc

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "GSoC: External Workspace Manager Plugin alpha version"
+tags:
+- pipeline
+- plugins
+- external-workspace-manager
+- gsoc
+author: alexsomai
+---
+
+Currently it's quite difficult to share and reuse the same workspace between multiple jobs and across nodes.
```

Comment:
You could just add a sentence to the end of the paragraph like "You can learn more about the design and goals of the External Workspace Manager project in _this introductory blog post_" ?